android/app/src/main/res/drawable/ic_account_circle_grey_24dp.xml

```diff
@@ -1,9 +1,5 @@
-<vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="56dp"
-    android:height="56dp"
-    android:viewportHeight="24.0"
-    android:viewportWidth="24.0">
-    <path
-        android:fillColor="#9b9b9b"
-        android:pathData="M12,2C6.48,2 2,6.48 2,12s4.48,10 10,10 10,-4.48 10,-10S17.52,2 12,2zM12,5c1.66,0 3,1.34 3,3s-1.34,3 -3,3 -3,-1.34 -3,-3 1.34,-3 3,-3zM12,19.2c-2.5,0 -4.71,-1.28 -6,-3.22 0.03,-1.99 4,-3.08 6,-3.08 1.99,0 5.97,1.09 6,3.08 -1.29,1.94 -3.5,3.22 -6,3.22z" />
+<vector android:height="24dp" android:viewportHeight="58.0"
```

Comment:
Don't change the existing vector please, reuse this instead

Reply:
@iamareebjamal Do I have to revert to previous vector used and shouldn't used another as stated in the issue?

Reply:
@iamareebjamal which vector to use now?

Reply:
The previous one is standard material vector drawable for profile, so let it be the default one

Reply:
@iamareebjamal Ok! Updating 👍